### PR TITLE
InputWithLabel

### PR DESCRIPTION
**Pull request by vitkon**
#### What's this PR do?
adds input with label and optional help text box

#### Where should the reviewer start?
`InputWithLabel.js`

#### How should this be manually tested?
`nwb serve`

#### What are the relevant tickets?
#20

### CODE REVIEW

File: src/components/atoms/HelpText.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes } from 'react';
```
Comment (michael-martin) [inline src/components/atoms/HelpText.js]: I like this syntax :)

File: src/components/atoms/HelpText.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+HelpText.getClasses = (props) => {
+    const prefix = 'dd';
+    const block = `${prefix}-${_.kebabCase(HelpText.displayName)}`;
+    const extraClasses = props.className && props.className.split(' ');
+
+    return cx({
+        [block]: true,
+        ..._.fromPairs(_.map(
+            extraClasses, className => [className, true]
+        ))
```
Comment (michael-martin) [inline src/components/atoms/HelpText.js]: How does this work @vitkon ? (Curiosity more than anything else :) ) I don't recognise some of the syntax here:

({
[block]: true,
..._.function....
})

?

Comment (vitkon) [inline src/components/atoms/HelpText.js]: 1. we come up with a new Object. I used object assign before.
2. [block] allows to use variable for the object key
3. and spread ... just composes other keys for this object (like in Object.assign)

Comment (vitkon) [inline src/components/atoms/HelpText.js]: 1. we come up with a new Object. I used `Object.assign` before.
2. `[block]` allows to use variable for the object key
3. and spread ... just composes other keys for this object (like in Object.assign). `_.fromPairs` eventually returns an Object.

File: src/components/molecules/CheckboxWithLabel.js [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
 
+import { mixin } from 'core-decorators';
+import ClassesMixin from '../../mixins';
+
 @bem({
-    block: 'dd-checkbox',
+    block: 'dd-checkbox-field',
     modifiers: ['disabled', 'required', 'readonly', 'checked']
 })
+@mixin(ClassesMixin)
```
Comment (vitkon) [inline src/components/molecules/CheckboxWithLabel.js]: @michael-martin mixins are here to stay ;)

**Pull request merged**